relay-kafka/src/statsd.rs

```diff
@@ -86,14 +86,14 @@ impl GaugeMetric for KafkaGauges {
     fn name(&self) -> &'static str {
         match self {
             KafkaGauges::InFlightCount => "kafka.in_flight_count",
```

Comment:
This one too?

Reply:
This one does not technically come out of `statistics`, so I'm undecided.

Reply:
ah good point, then I agree it should not have the mid-fix